packages/sdk/src/__tests__/engine-commands/getDmmf.test.ts

```diff
@@ -143,28 +143,27 @@ describe('getDMMF', () => {
     try {
       await getDMMF({ datamodel })
     } catch (e) {
-      const snapshot = `Get DMMF: Schema parsing
-error: Error parsing attribute \"@default\": The \`autoincrement()\` default value is used on a non-id field even though the datasource does not support this.
-  -->  schema.prisma:7
-   | 
- 6 |       model User {
- 7 |         id        Int      @default(autoincrement())
- 8 |         email     String   @unique
-   | 
-error: Error parsing attribute \"@default\": The \`autoincrement()\` default value is used on a non-indexed field even though the datasource does not support this.
-  -->  schema.prisma:7
-   | 
- 6 |       model User {
- 7 |         id        Int      @default(autoincrement())
- 8 |         email     String   @unique
-   | 
-
-Validation Error Count: 2`
-      if (process.env.PRISMA_CLI_QUERY_ENGINE_TYPE === 'binary') {
-        expect(stripAnsi(e.message)).toEqual(`${snapshot}\n`)
-      } else {
-        expect(stripAnsi(e.message)).toEqual(snapshot)
-      }
+      expect(stripAnsi(e.message)).toMatchInlineSnapshot(`
+        "Get DMMF: Schema parsing
+        error: Error parsing attribute \\"@default\\": The \`autoincrement()\` default value is used on a non-id field even though the datasource does not support this.
```

Comment:
It is, that's automatically generated by jest when using `toMatchInlineSnapshot`. Previously, the binary had an additional newline character that prevented us from using such method directly, and forced us to use a conditional `toEqual`.

Reply:
My understanding is that the binary already had an additional newline w.r.t. the library, but that wasn't captured by tests previously, as only the query-engine library was snapshotted. Joel and I knew that there was an additional newline either in the Rust codebase or in the TypeScript one. Since it wasn't a blocker, we merged it with those conditional tests. Now, as I had to slightly touch the getDmmf file again, I have spotted that additional "\n" in the binary and got rid of it.